tests/test_integration.py

```diff
@@ -141,20 +141,32 @@ def test_regular_run_native_toxenv():
         assert len(list(sitelib.glob(f"{pkg}-*.dist-info"))) == 1
 
 
-def test_regular_after_current_is_not_supported():
+def test_regular_after_current_is_supported():
     result = tox("-e", NATIVE_TOXENV, "--current-env")
     assert result.stdout.splitlines()[0] == NATIVE_EXECUTABLE
+    result = tox("-e", NATIVE_TOXENV, prune=False)
+    assert f"/.tox/{NATIVE_TOXENV}/bin/python" in result.stdout
+    assert "--recreate" not in result.stderr
+
+
+def test_regular_after_killed_current_is_not_supported():
+    # fake broken tox run
+    shutil.rmtree(DOT_TOX, ignore_errors=True)
+    (DOT_TOX / NATIVE_TOXENV / "bin").mkdir(parents=True)
+    (DOT_TOX / NATIVE_TOXENV / "bin" / "python").symlink_to(NATIVE_EXECUTABLE)
+
     result = tox("-e", NATIVE_TOXENV, prune=False, check=False)
     assert result.returncode > 0
-    assert "not supported" in result.stderr
+    assert "--recreate" in result.stderr
 
 
-def test_regular_after_first_deps_only_is_not_supported():
+def test_regular_after_first_deps_only_is_supported():
     result = tox("-e", NATIVE_TOXENV, "--print-deps-only")
     assert result.stdout.splitlines()[0] == "six"
-    result = tox("-e", NATIVE_TOXENV, prune=False, check=False)
-    assert result.returncode > 0
-    assert "not supported" in result.stderr
+    result = tox("-e", NATIVE_TOXENV, prune=False)
+    lines = sorted(result.stdout.splitlines()[:1])
+    assert "--recreate" not in result.stderr
+    assert f"/.tox/{NATIVE_TOXENV}/bin/python" in lines[0]
```

Comment:
The sorted lines and this look a bit flimsy to me. Wouldn't it be safer to search in all the `lines` instead?

Reply:
It's a pattern copied form other tests. If we don't want it, we can fix all of them in a later PR.

Reply:
Ah, I see. In that case I would let it in. There's probably a good reason for it that I don't have enough insight to see.

Reply:
Or maybe it isn't :D All I'm saying is that it should probably not block this PR.